src/ontology/mondo-edit.obo

```diff
@@ -24212,7 +24212,7 @@ is_a: MONDO:0000942 {source="DOID:1237"} ! corneal disorder
 [Term]
 id: MONDO:0001516
 name: spinal muscular atrophy
-def: "Spinal muscular atrophy (SMA) refers to a group of inherited conditions that affect the muscles. The severity of the condition; the associated signs and symptoms; and the age at which symptoms develop varies by subtype. In general, people with SMA experience progressive weakness and atrophy of muscles involved in mobility, the ability to sit unassisted, and head control. Breathing and swallowing may also be affected in severe cases. SMA is generally caused by changes (mutations) in the SMN1 gene and is inherited in an autosomal recessive manner. Extra copies of the SMN2 gene modify the severity of SMA. Rare autosomal dominant (caused by mutations in DYNC1H1, BICD2, or VAPB genes) and X-linked (caused by mutations in UBA1) forms of SMA exist. Treatment is based on the signs and symptoms present in each person." [https://rarediseases.info.nih.gov/diseases/7674/spinal-muscular-atrophy]
+def: "A motor neuron disease that affect the muscles, and characterized by muscle weakness and atrophy resulting from progressive degeneration and irreversible loss of the anterior horn cells in the spinal cord (i.e., lower motor neurons) and the brain stem nuclei.The severity of the condition; the associated signs and symptoms; and the age at which symptoms develop varies by subtype. In general, people with SMA experience progressive weakness and atrophy of muscles involved in mobility, the ability to sit unassisted, and head control. Breathing and swallowing may also be affected in severe cases. SMA is generally caused by changes (mutations) in the SMN1 gene and is inherited in an autosomal recessive manner. Extra copies of the SMN2 gene modify the severity of SMA. Rare autosomal dominant (caused by mutations in DYNC1H1, BICD2, or VAPB genes) and X-linked (caused by mutations in UBA1) forms of SMA exist. Treatment is based on the signs and symptoms present in each person." [GARD:7674, https://orcid.org/0000-0001-6330-7526, PMID:20301526]
```

Comment:
FYI- the abbreviation `SMA` is used in the definition before it is defined, e.g. spinal muscular atrophy (SMA)